json-endpoints/class.wpcom-json-api-site-settings-endpoint.php

```diff
@@ -424,6 +426,26 @@ public function update_settings() {
 					}
 					break;
 
+				case 'advanced_seo_front_page_description':
+					$front_page_description = sanitize_text_field( $value );
+
+					// The seo front page meta description should be shorter than 300 characters
+					$front_page_description = mb_substr( $front_page_description, 0, 300 );
```

Comment:
I think it would be interesting to add a filter here, so folks can shorten the number of characters if needed.